docs/content/preview/api/ycql/ddl_drop_type.md

```diff
@@ -51,4 +51,3 @@ ycqlsh:example> DROP TYPE person;
 ## See also
 
 - [`CREATE TABLE`](../ddl_create_table)
-- [`DROP TYPE`](../ddl_drop_keyspace)
```

Comment:
The link is for DROP KEYSPACE. You need to change the title to [`DROP KEYSPACE`](../ddl_drop_keyspace) everywhere

Reply:
Done.